.ci/common/test.sh

```diff
@@ -110,7 +110,16 @@ run_oldtests() {
 }
 
 run_single_includes_tests() {
-  ${MAKE_CMD} -C "${BULID_DIR}" check-single-includes
+  ulimit -c unlimited
+  if ! ${MAKE_CMD} -C "${BUILD_DIR}" check-single-includes; then
+    asan_check "${LOG_DIR}"
+    valgrind_check "${LOG_DIR}"
+    check_core_dumps
```

Comment:
Confirmed the tests seem to pass with ulimit -c unlimited without asan_check, valgrind_check, and check_core_dumps.
Is "ulimit -c" necessary in this case?